Cannot find module 'lib/rsvp'

Might be user error. Happens in 1.1.1 and 1.2.0.

Installing rsvp via npm. Checking inside of the node_modules/rsvp installation where index.js is located and it calls

``` javascript
module.exports = require('lib/rsvp');
```

There is no lib directory in that folder.

Is there another way I should be loading this other than

``` javascript
var RSVP = require( 'rsvp' );
```

Error print out is below

``` bash
module.js:337
    throw new Error("Cannot find module '" + request + "'");
          ^
Error: Cannot find module 'lib/rsvp'
    at Function._resolveFilename (module.js:337:11)
    at Function._load (module.js:279:25)
    at Module.require (module.js:359:17)
    at require (module.js:375:17)
    at Object.<anonymous> (/Users/jlembeck/code/image-resize-test/node_modules/rsvp/index.js:1:80)
    at Module._compile (module.js:446:26)
    at Object..js (module.js:464:10)
    at Module.load (module.js:353:31)
    at Function._load (module.js:311:12)
    at Module.require (module.js:359:17)
```

--- a/.npmignore
+++ b/.npmignore
@@ -2,7 +2,6 @@
 /.npmignore
 /.bundle
 /component.json
-/index.js
 /bin
 /browser/
 /node_modules/

--- a/index.js
+++ b/index.js
@@ -1,1 +1,0 @@
-module.exports = require('lib/rsvp');

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "rsvp",
   "version": "1.2.0",
   "description": "A lightweight library that provides tools for organizing asynchronous code",
-  "main": "lib/rsvp.js",
+  "main": "node/rsvp.js",
   "directories": { "lib": "lib" },
   "devDependencies": {
     "jshint": "~0.9",

